assets/js/validate.js

```diff
@@ -0,0 +1,40 @@
+const formData = {};
+const appForms = document.forms[0];
+const inputFields = document.querySelectorAll('.userdata');
```

Comment:
- [ **OPTIONAL** ] Good job doing all the changes required by the previous reviewer. But I think it would be better if you move the `assets/js` folder in the root directory and delete other unused javascript files to make your project clean as all codes in other files are duplicated with code in the `assets/js` files.